gensim/models/normmodel.py

```diff
@@ -13,34 +13,58 @@
 
 class NormModel(interfaces.TransformationABC):
     """
-    Objects of this class realize the explicit normalization of
-    vectors. Supported norms are l1' and 'l2' with 'l2' being
-    default.
-
-    The main methods are:
-
-    1. Constructor which normalizes the terms in the given corpus document-wise.
-    2. The normalize() method which normalizes a simple count representation.
-    3. The [] transformation which internally calls the self.normalize() method.
+    Objects of this class realize the explicit normalization of vectors.
+
+    Model persistency is achieved via its load/save methods.
+
+    Parameters
+    ----------
+    corpus : iterable
+        Iterable of documents.
+    norm : {'l1', 'l2'}
+        Norm used to normalize. l1 and l2 norms are supported (l2 is default)
+
+    Methods
```

Comment:
No need for `Methods` section, there aren't so many methods here :)

Reply:
OK